features/pakiti/client/configv3.pan

```diff
@@ -0,0 +1,37 @@
+unique template features/pakiti/client/configv3;
+
+variable PAKITI_RPMS ?= if_exists('config/pakiti/client/config');
```

Comment:
As with another open PR, I'd suggest a more pragmatic approach as this PR really improves the existing situation (Pakiti v3 support is important). Merge it and open an issue assigned for 14.6 to move the config to metaconfig.